embarrassingly-parallel.ipynb

```diff
@@ -160,10 +161,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Use Dask Delayed to make our function lazy\n",
+    "## Use [Dask Delayed](http://dask.pydata.org/en/latest/delayed.html) to make our function lazy\n",
     "\n",
     "We can call `dask.delayed` on our funtion to make it lazy.  Rather than compute its result immediately, it records what we want to compute as a task into a graph that we'll run later on parallel hardware.\n",
     "\n",
+    "Using `dask.delayed` is a relatively straightforward way to parallelize an existing code base, even if the computation isn't embarrassingly parallel like this one. It's less useful if you need to be calling delayed functions from other delayed functions (the Futures API is better for that kind of workload). See [delayed best practices](http://dask.pydata.org/en/latest/delayed-best-practices.html) for more on using dask.delayed effectively.\n",
```

Comment:
> It's less useful if you need to be calling delayed functions from other delayed functions (the Futures API is better for that kind of workload). See [delayed best practices](http://dask.pydata.org/en/latest/delayed-best-practices.html) for more on using dask.delayed effectively

I recommend that we skip this statement for this example.  It's a good point, but I suspect that more beginning readers are likely to be confused by this than gain benefit from it.  They may not understand what it means to call a delayed function from within a delayed function.  Also, this is just as doable/problematic with dask.delayed as calling futures from within futures code.  Both are doable, though neither is particularly recommended.

Reply:
I'm fine with this, done.